packages/muelu/doc/UsersGuide/options.tex

```diff
@@ -215,7 +215,10 @@ \section{Aggregation options}
                                     This means the coarse ratio is higher than in \verb!mis2 aggregation!.
                                     Faster than coloring-based aggregation.  Cannot control min/max aggregate size. & Yes & Yes \\
       \verb!serial! &               Computes a D2 coloring on host, and then does 4-phase aggregation on device. & No & Optional \\
-      \verb!vertex based bit set! & Computes a D2 coloring in parallel with conflicts resolved by neighbors-of-neighbors loop. Then does 4-phase aggregation on device. & Yes & No \\
+      \verb!default! &              Computes the same as \verb!net based bit set!. & No & Optional \\
```

Comment:
Maybe we're not looking at the same documentation, but this seems also wrong.

Reply:
I see it now. It's both, not one or the other